app/models/application_record.rb

```diff
@@ -0,0 +1,3 @@
+class ApplicationRecord < ActiveRecord::Base
```

Comment:
c'est la nouvelle syntaxe de Rails 5 --> http://blog.bigbinary.com/2015/12/28/application-record-in-rails-5.html